WVAanalysisRun2/src/setInputTree.cc

```diff
@@ -65,6 +65,8 @@ void setInputTree::Init()
    fChain->SetBranchAddress("Weight", &Weight, &b_Weight);
    fChain->SetBranchAddress("PUWeight", &PUWeight, &b_PUWeight);
    fChain->SetBranchAddress("genEventWeight", &genEventWeight, &b_genEventWeight);
+   fChain->SetBranchAddress("originalWeight", &originalWeight, &b_originalWeight);
+   fChain->SetBranchAddress("AQGCweights", &AQGCweights, &b_AQGCweights);
```

Comment:
Need to change to:
fChain->SetBranchAddress("GenEventInfoAQGCweights", &AQGCweights, &b_AQGCweights);